cmd/tke-installer/app/installer/manifests/tke-registry-api/tke-registry-api.yaml

```diff
@@ -73,7 +73,8 @@ spec:
             requests:
               cpu: 50m
               memory: 128Mi
-      nodeName: {{ .NodeName }}
+      nodeSelector:
```

Comment:
if hostnameAsNodename is true, then the node name is hostname here, so `nodeName` still works, why use `nodeSelector` instead ? `nodeName` will not go through `scheduler` logic rather than `nodeSelector`